src/scribe_data/check/check_project_metadata.py

```diff
@@ -37,13 +21,6 @@
 
 
 def get_available_languages() -> dict[str, list[str]]:
-    """
-    Get available languages from the data extraction folder.
```

Comment:
Quick note here @Deadpool2000 that for this and the next function their docstrings were removed :) No stress! I'll add them back in 😊